### DISCUSSION TITLE
How do I enable the x and y coordinates of the mouse cursor on the screen?

### DISCUSSION THREAD

**Discussion by Zalo2** [ScratchAddons/ScratchAddons#5330] [Help] [answered]
When I watch tutorials on Scratch, sometimes I see an x and y coordinate indicator at the top left of the stage next to the flag and stop buttons. I've checked some of the addons, and I haven't found this one. How do I enable it?

**Comment by Samq64** [comment, marked as answer]
https://scratch.mit.edu/scratch-addons-extension/settings#addon-mouse-pos

**Reply by Zalo2** [reply]
Thanks!
